app/controllers/application_controller.rb

```diff
@@ -2,11 +2,19 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
-  
```

Comment:
Where was this whitespace from originally? The new stuff in this PR looks clean, but if there's a setting on your test editor that can take care of trailing whitespace for you, we should definitely set it up.